docs/installation.rst

```diff
@@ -1,3 +1,116 @@
 Installation
 ============
 
+**System Requirements**
+
++ 64-bit Architecture
++ A minimum 250GB storage and 4GB memory
++ Developed and tested on Fedora 27
+
+**Pre-requisites**
+
++ All system should have Libvirt API installed for VM provisioning.
+
+::
+
+    dnf install qemu-kvm qemu-img virt-manager libvirt libvirt-python libvirt-client virt-install -y
+
+
++ All compute resources should have Docker installed for running Docker containers. To install Docker on Fedora  `follow <https://docs.docker.com/install/linux/docker-ce/fedora/#install-using-the-repository>`_
+
++ Server should have Docker machine installed.
+
+::
+
+    curl -L https://github.com/docker/machine/releases/download/v0.13.0/docker-machine-`uname -s`-`uname -m` >/tmp/docker-machine && sudo install /tmp/docker-machine /usr/local/bin/docker-machine
+
+
++ Server should have SSH public key or it can be generated using SSH keygen.
+
+::
+
+    ssh-keygen
+
+
+**Installation**
+
+*From Docker*
+
+To run Minisat using docker, first clone the Github Minisat repository using and go to directory Minisat
```

Comment:
Make this "To containerize Minisat, first clone the Github Minisat repository using and go to directory Minisat"